meteor/client/menu.html

```diff
@@ -12,40 +12,64 @@
             <ul>
                 <li><img class="mozillalogo" src="/images/mozilla.svg" ></li>
                 <li><a class="mozdef" href="/" title="MOZDEF">MOZDEF</a></li>
-                <li> <a target="_blank" href={{mozdef.kibanaURL}}>Kibana</a>
-                    <ul>
-                        <li class="first">
-                            {{#each kibanadashboards as item}}
-                                {{>kibanaDashboardItem}}
-                            {{/each}}
-                        </li>
-                    </ul>
-                </li>
-                <li><a href="/alerts/">Alerts</a>
-                    <ul>
-                        <li class="first"><a href="/ipblocklist">ip blocklist</a></li>
-                        <li class="first"><a href="/fqdnblocklist">fqdn blocklist</a></li>
-                    </ul>
-                </li>
-                <li><a href="/investigations">Investigations</a>
-                    <ul>
-                        <li class="first"><a href="/investigation/new/">new investigation</a></li>
-                    </ul>
-                </li>
-                <li><a href="/incidents">Incidents</a>
-                    <ul>
-                        <li class="first"><a href="/incident/new/">new incident</a></li>
-                        <li class="last"><a href="/incidents/veris/">veris stats</a></li>
-                    </ul>
-                </li>
-                <li class="first"><a href="#">Visualizations</a>
-                    <ul>
-                        <li class="first"><a href="/logincounts/">logincounts</a></li>
-                        <li><a href="/attackers">attackers</a></li>
-                        <li class="last"><a href="/globe">globe</a></li>
-                    </ul>
-                </li>
-                <li><a href="/about">About</a></li>
+                {{#if isFeature "kibana"}}
```

Comment:
One weird thing that I've noticed, is if you turn off a few features via the environment variable, and then rebuild meteor, everytime you refresh the page (or originally load it), the disabled features links originally appear, and then disappear. 

Is there a way to fix this?

Reply:
I fought that for a while. The issue is the subscription to the features table is ready before records show up (this is universal to all collections and is just the async way meteor works). Since the 'menu' template is reactive it removes the features once the data shows up; but initially it's empty so everything shows. 
I'll keep looking for more elegant ways to avoid showing the menu until we have records; but I don't know of one currently.

Reply:
Just had a brainstorm and added a utility function the template can use to know when records are ready. It now delays painting until it knows the features.